app/serializers/user_notification_total_serializer.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+class UserNotificationTotalSerializer < ApplicationSerializer
+  attributes :id,
+             :username,
+             :unread_notifications,
+             :unread_personal_messages,
+             :unseen_reviewables,
+             :topic_tracking,
+             :group_inboxes
+
+  def unread_notifications
+    object.all_unread_notifications_count - object.new_personal_messages_notifications_count
```

Comment:
We need to memoize `object.new_personal_messages_notifications_count` to avoid executing the same query twice.